examples/grpc_dynamic_codegen/setup.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const opentelemetry = require('@opentelemetry/core');
+const { NodeTracer } = require('@opentelemetry/node-sdk');
+const { SimpleSpanProcessor } = require('@opentelemetry/tracer-basic');
+const { JaegerExporter } = require('@opentelemetry/exporter-jaeger');
+const { ZipkinExporter } = require('@opentelemetry/exporter-zipkin');
+const EXPORTER = process.env.EXPORTER || '';
+
+function setupTracerAndExporters(service) {
+  const tracer = new NodeTracer({
+    plugins: {
+      grpc: {
+        enabled: true,
+        // You may use a package name or absolute path to the file.
+        path: '@opentelemetry/plugin-grpc'
+      }
+    }
+  });
+
+  let exporter;
+  if (EXPORTER.toLowerCase().startsWith('z')) {
+    exporter = new ZipkinExporter({
+      serviceName: service,
+    });
+  } else {
+    exporter = new JaegerExporter({
+      serviceName: service,
+      // The default flush interval is 5 seconds.
+      flushInterval: 2000
+    });
+  }
+
+  tracer.addSpanProcessor(new SimpleSpanProcessor(exporter));
```

Comment:
Should we add a comment that typically for production you should use `BatchSpanProcessor`?

This gets back to my suggestion that we just have ZipkinExporter and JaegerExporter compose together a `BatchSpanProcessor` and say a `ZipkinWriter` or similar. That way people don't need to worry about this composition on their own. My feeling is that people tend to copy/paste from examples.